ivadomed/transforms.py

```diff
@@ -129,7 +134,7 @@ def __call__(self, sample):
             roi_data = sample['roi']
             rdict['roi'] = []
             for roi in roi_data:
-                rdict['roi'] = self.resample_bin(roi, wshape_new, hshape_new, 0.0)
+                rdict['roi'].append(self.resample_bin(roi, wshape_new, hshape_new, 0.0))
```

Comment:
The `thr` of `resample_bin` is not used anymore.
We may want to binarise during the "do transform" to keep a binary GT, but not mandatory, to discuss and check with the loss functions.
We do not want to binarise during the "undo transform".